contrib/debian/copyright

```diff
@@ -75,12 +77,15 @@ Copyright: Stephen Hutchings, Jonas Schnelli
 License: Expat
 Comment: Modifications of Stephen Hutchings' Typicons
 
-Files: src/qt/res/icons/bitcoin.*
-       share/pixmaps/bitcoin*
-       src/qt/res/src/bitcoin.svg
-Copyright: Bitboy, Jonas Schnelli
-License: public-domain
-Comment: Site: https://bitcointalk.org/?topic=1756.0
+Files:
+ src/qt/res/icons/vertcoin.*
+ share/pixmaps/vertcoin*
+ src/qt/res/src/vertcoin.svg
+Copyright:
+ Bushido
+License: CC-BY-SA-4.0
+Comment:
+ https://commons.wikimedia.org/wiki/File:Vertcoin.svg
```

Comment:
While we're modifying this file, lets make the formatting the same as what's around it (without extra lines)
I know... i created this formatting variance before haha